src/components/table/column-picker/column-picker.component.html

```diff
@@ -22,19 +22,22 @@
         <!-- Create item for not expandable node -->
         <cdk-tree-node *cdkTreeNodeDef="let node"
             [attr.aria-hidden]="selected && selected.indexOf(node.name) > -1">
-            <div uxTabbableListItem
-                [uxSelectionItem]="node.column"
-                *ngIf="_shouldRenderNode(node)"
-                class="column-picker-list-item"
-                [ngClass]="'column-picker-tree-node-level-' + node.level">
-
-                <ng-container *ngIf="!deselectedTemplate">{{ node.name }}</ng-container>
-
-                <ng-container
-                    *ngIf="deselectedTemplate"
-                    [ngTemplateOutlet]="deselectedTemplate"
-                    [ngTemplateOutletContext]="{ $implicit: node.name }">
-                </ng-container>
+            <div role="group">
```

Comment:
This group will only ever contain one item. Since this is set up as a flat tree, which you can observe by looking at the DOM:
![image](https://user-images.githubusercontent.com/26114101/217784497-648da136-3809-412a-9199-1589f65d710b.png)

you are not able to use `role="group"` as intended. With a screen reader, every item will be `item one of one`.

You also added `role="treeitem"` which duplicates the role of the parent element.

I suspect that the fix is to move `aria-selected` to the cdk-tree-node element.